src/scss/components/_button.scss

```diff
@@ -26,14 +26,18 @@ $button-padding-h: $grid-margin-2;
   $color-active: null,
   $color-background-active: null,
   $color-border-active: null,
-  $force-colors: false
+  $force-colors: false,
+  $color-border-focus: null,
+  $color-border-focus-visible: null
```

Comment:
I don't see any usage of this variable, let's remove it as the same styles are applied to `:focus` and `:focus-visible` now anyway.